### PR TITLE
Projection support test only

### PR DESCRIPTION
**Pull request by dnachev**
This is the projection test from #54 extracted in a standalone PR and all tests skipped.

### CODE REVIEW

**Comment by dnachev** [comment]
I'm not quite sure why the tests fails - it is complaining about the prettier format of the `projection-test.js`, but they are passing locally and reformatting the file does not yield any differences.

File: tests/helpers/watch-property.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      return propertyNames.reduce((result, prop) => {
+        result[prop] = watchers[prop].count;
+        return result;
+      }, {});
+    },
+    unwatch,
+  };
+}
+
+QUnit.assert.dirties = function assertDirties(options, updateMethodCallback, label) {
```
Comment (hjdivad) [inline tests/helpers/watch-property.js]: should probably be something like `expectDirty` no?

Comment (dnachev) [reply tests/helpers/watch-property.js]: This follows the QUnit `assert.throws()` pattern.

**Comment by dnachev** [comment]
Closed it unintentionally.

**Comment by dnachev** [comment]
OK, I've deleted the base branch, which caused this to close automatically.

**Pull request merged**